conduct.md

```diff
@@ -134,14 +134,14 @@ TODO: Write an introductory paragraph.
 
 TODO: Write an introductory paragraph.
 
-#### Acceptable Conduct
+#### Level 1 Infraction -- point 1
 
-- TODO
-- TODO
-- TODO
+- Students who get 1 point will be recorded by professor and warned orally.
```

Comment:
@Yanqiao4396, I left the following comment on @gvanzin-allegheny's PR #6  and I'd like to get your thoughts on my suggestion as well.

> Since each violation corresponds to one point, might this infraction policy be easier to understand if the point system was removed altogether? For instance, anywhere it refers to a student receiving a point could be reworded to say that the student has committed a violation.